samples/Actor/ActorClient/ActorClient.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
   </ItemGroup>
```

Comment:
This can be removed as its not used

Reply:
The dependency of Newtonsoft.Json is derived from the original code, of course I can remove it.